src/test/java/co/rsk/federate/signing/hsm/advanceblockchain/ConfirmedBlocksProviderTest.java

```diff
@@ -21,8 +22,8 @@
 
 class ConfirmedBlocksProviderTest {
 
-    private final int HSM_VERSION_2 = 2;
-    private final int HSM_VERSION_4 = 4;
+    private static final int HSM_VERSION_2 = HSMVersion.V2.getNumber();
+    private static final int HSM_VERSION_4 = HSMVersion.V4.getNumber();
```

Comment:
I'd remove these veriables and use the enum values directly

Reply:
Done